plugins/admin/pkg/command/profiling/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// DO NOT COMMIT
```

Comment:
"DO NOT COMMIT" and it's in the PR!! :)

Reply:
This is just sample code for reference as I need to co-work with @chaozbj. This file will NOT exist in the final PR.